wordle-web/src/views/WordleView.vue

```diff
@@ -1,23 +1,71 @@
 <template>
-  <h1>Wordle Mind Bender</h1>
+  <main class="align-center">
+    <br />
+    <h1>The Good Word</h1>
+    <br />
 
-  <GameBoard :game="game" @letterClick="addChar" />
+    <div>
+      <GameBoard :game="game" @letterClick="addChar" />
+      <br />
```

Comment:
Could use padding or margin for this possibly, not positive where br should be used.

Reply:
I like how the new line (br) adds space between the game board and the components below. We will play around with using padding and/or margins to see what we can do with it.